src/main/java/by/custom/utilcalculator/domain/tree/HelperTree.java

```diff
@@ -75,4 +212,12 @@ private static void fillParents(final Node node) {
             }
         }
     }
+
+    public static Map<Command, StepsIndicator> fillCommandsToFields(Map<StepsIndicator, Command> fieldsToCommands) {
+        Map<Command, StepsIndicator> commandsToFields = new HashMap<>();
```

Comment:
Looks like it should be `Map<Command, List<StepsIndicator>>`. Please, look at `Command.ENGINE_VOLUME`.

Reply:
I see - we did a mistake using the same key for different values. I'll fix it easily.